docs/gaps/gaps.tex

```diff
@@ -0,0 +1,286 @@
+\documentclass{article}
+
+%%%% packages and definitions (optional)
+\usepackage{graphicx} % allows inclusion of graphics
+\usepackage{graphics}
+\usepackage{placeins}
+\usepackage{booktabs} % nice rules (thick lines) for tables
+\usepackage{microtype} % improves typography for PDF
+\usepackage{xspace}
+\usepackage[hidelinks]{hyperref}
+\usepackage{xspace}
+\usepackage{hhline}
+
+\usepackage{tabularx}
+\newcolumntype{b}{X}
+\newcolumntype{s}{>{\hsize=.5\hsize}X}
+\newcolumntype{m}{>{\hsize=.75\hsize}X}
+
+\newcommand{\SN}{S$_N$}
+\renewcommand{\vec}[1]{\bm{#1}} %vector is bold italic
+\newcommand{\vd}{\bm{\cdot}} % slightly bold vector dot
+\newcommand{\grad}{\vec{\nabla}} % gradient
+\newcommand{\ud}{\mathop{}\!\mathrm{d}} % upright derivative symbol
+\newcommand{\Cyclus}{\textsc{Cyclus}\xspace}%
+\newcommand{\Cycamore}{\textsc{Cycamore}\xspace}%
+\graphicspath{ {images/} }
+\usepackage[affil-it]{authblk}
+\usepackage{tikz}
+\usetikzlibrary{positioning, arrows, decorations, shapes }
+\usepackage{cleveref}
+
+
+\usepackage[acronym,toc]{glossaries}
+\include{acros.tex}
```

Comment:
usually having the .tex here breaks things (many distributions of latex will look for "acros.tex.tex")